binaries/.classpath_cocoa

```diff
@@ -34,8 +34,8 @@
 	<classpathentry kind="src" path="Eclipse SWT WebKit/cocoa"/>
 	<classpathentry kind="src" path="Eclipse SWT OpenGL/cocoa"/>
 	<classpathentry kind="src" path="Eclipse SWT OpenGL/common"/>
-	<classpathentry kind="lib" path="lib/skija-shared-0.116.2.jar"/>
-	<classpathentry kind="lib" path="lib/skija-macos-arm64-0.116.2.jar"/>
+	<classpathentry kind="lib" path="lib/skija-macos-arm64-0.116.4.jar"/>
+	<classpathentry kind="lib" path="lib/skija-shared-0.116.4.jar" sourcepath="lib/skija-shared-0.116.4-sources.jar"/>
 	<classpathentry kind="lib" path="lib/types-0.1.1.jar"/>
```

Comment:
Sorry for the comment after already approving: I think the typed library is missing a reference to the added sources (e.g., classes like `IPoint` are not available as sources now). In addition, the OS-specific JARs do not seem to contain any relevant source code (just DLLs), so we could probably leave out those DLLs?